### PR TITLE
Shift urgency integer values up by one

### LINKED ISSUE #78: Consider changing urgency range to 0-7
Current range is -1 to 6, some are in preference of 0-7 (CC @ianswett)

### PR DESCRIPTION
**Pull request by rmarx**
Fixes #78

### CODE REVIEW

File: draft-kazuho-httpbis-priority.md [hunk 1]
```diff
@@ -246,47 +246,49 @@ Unknown parameters MUST be ignored.
...
-| between 1 and 5 | supplementary ({{supplementary}}) |
-|               6 | background ({{background}})       |
+|               0 | prerequisite ({{prerequisite}})   |
+|               1 | default ({{default}})             |
+| between 2 and 6 | supplementary ({{supplementary}}) |
+|               7 | background ({{background}})       |
 {: #urgencies title="Urgencies"}
 
-The value is encoded as an sh-integer.  The default value is zero.
+The value is encoded as an sh-integer. The default value is zero.
```
Comment (ianswett) [inline draft-kazuho-httpbis-priority.md]:
```suggestion
The value is encoded as an sh-integer. The default value is 1.
```

**Review by ianswett — CHANGES_REQUESTED** [review]
LG, one spot I think you missed(because it was zero, not 0)

File: draft-kazuho-httpbis-priority.md [hunk 2]
```diff
@@ -731,6 +733,10 @@ Mike Bishop, Roberto Peon, Robin Marx, Roy Fielding.
 
 # Change Log
 
+## Since draft-kazuho-httpbis-priority-03
+
+* Changed numbering from [-1,6] to [0,7] (#78)
```
Comment (LPardue) [inline draft-kazuho-httpbis-priority.md:738]: Let's be more explicit and say something like "urgency parameter value range shifted to zero-base-index ( from [-1,6] to [0,7]).

File: draft-kazuho-httpbis-priority.md (hunk 1 above)
Comment (rmarx) [reply draft-kazuho-httpbis-priority.md]: Ah yes, thanks for spotting that!

File: draft-kazuho-httpbis-priority.md (hunk 2 above)
Comment (LPardue) [inline draft-kazuho-httpbis-priority.md:738]:
```suggestion
* Urgency parameter value range shifted to zero-base index
   (from [-1,6] to [0,7]) (#78)
```

**Review by LPardue — CHANGES_REQUESTED** [review]
Minor nit in the changelog but otherwise LGTM

File: draft-kazuho-httpbis-priority.md (hunk 2 above)
Comment (LPardue) [reply draft-kazuho-httpbis-priority.md:738]: I'll apply this manually in an editorial cleanup.

Comment (rmarx) [reply draft-kazuho-httpbis-priority.md:738]: (y) thanks

**Pull request merged**